source/Thumb.js

```diff
@@ -83,11 +98,11 @@
 				}
 				var sbo = inStrategy[this.positionMethod]() - over;
 				// calc size & position
-				var bdc = bd - this.cornerSize;
+				var bdc = bd - moon.riScale(this.cornerSize);
```

Comment:
Do these (`cornerSize` and `minSize`) need to happen on `update()` or can they be precalculated on `create()`?